### PR TITLE
Updated check-eligibility.js to run checks and move to next view.

### PR DESCRIPTION
**Pull request by jasnakai**
I want to check to make sure the solution is valid before moving on to the next steps.

### CODE REVIEW

**Review by heymatthenry — CHANGES_REQUESTED** [review]
This looks good so far! It needs a test though. Check out the [UEI validation spec](https://github.com/GSA-TTS/FAC-Frontend/blob/main/cypress/integration/check-ueid.spec.js#L78-L152) for an example of how to intercept & mock API calls.

**Comment by jasnakai** [comment]
I added the Cypress tests for submitting the form and receiving the response, then navigating to the new page on success.
I had a couple of questions:
Is there an error message that has been designed for a failure response from the form submission?
On line 14/15 of check-eligibility.spec.js, there is a command to throw an error if the form is submitted. It was being triggered by the later form submission tests.
I commented out the line for now. Not sure if this is correct or how to handle this properly.

**Comment by heymatthenry** [comment]
@jasnakai There was one spec in there that I marked as `skipped` because I didn't have time to debug it when it started behaving differently than I expected, but I noticed [you set it to not be skipped](https://github.com/GSA-TTS/FAC-Frontend/pull/22/commits/51066c1b35e4db1bccc59e54f2b1e52a16f5fb70#diff-db94b8b630415163544bab9773f821a793a02e6d9316ec61d3e8703043980229R57). Does the error you're seeing get thrown during the new specs you're adding or is it just during the one that was previously skipped? If it's the former we should get to the bottom of it. If the latter, please just skip that spec again for now (i.e. change the `it` back to `xit`)

File: cypress/integration/check-eligibility.spec.js [hunk 1]
```diff
@@ -62,6 +63,48 @@ describe('Create New Audit', () => {
...
+  describe('Eligibility validation via API', () => {
+    it('should return eligibility errors from the remote server', () => {
+      cy.intercept(
+        {
+          method: 'POST',
+          url: '/sac/eligibility',
+        },
+        {
+          elegible: false,
+          errors: 'Not elegible.',
```
Comment (heymatthenry) [inline cypress/integration/check-eligibility.spec.js:75]: Very small thing, but please change "elegible" to "eligible" throughout this spec.

Comment (jasnakai) [reply cypress/integration/check-eligibility.spec.js:75]: Will do on the elegible to eligible. As for the update to the skipped spec, I accidentally updated it and then reverted it in my last commit.

Comment (jasnakai) [reply cypress/integration/check-eligibility.spec.js:75]: The error that gets thrown is from the Blank Form test block, specifically the 'will not submit' test. It is not a Cypress error but an error specified in our test script. I wonder if I change my new tests from a click on the Continue button to a form submit with the e.preventDefault added to the commands.

**Pull request merged**